playbooks/robusta_playbooks/pod_enrichments.py

```diff
@@ -13,12 +15,39 @@ def pod_graph_enricher(pod_event: PodEvent, params: ResourceGraphEnricherParams)
         "pod": pod_event.get_pod().metadata.name,
         "namespace": pod_event.get_pod().metadata.namespace,
     }
+    pod = pod_event.get_pod()
+    limit_lines = []
+    resource_limits = pod_limits(pod)
```

Comment:
Would it make more sense to do it:

if params.display_limits:
  resource_limits = pod_limits
  if params.resource_type == "CPU" ...

Reply:
do you mean:
```
if params.display_limits:
     resource_limits = pod_limits
     if params.resource_type == "CPU" and resource_limits.cpu > 0:
```

Reply:
yeap